packages/neon/neon/lib/src/blocs/apps.dart

```diff
@@ -192,4 +192,10 @@ class AppsBloc extends InteractiveBloc implements AppsBlocEvents, AppsBlocStates
 
   List<Provider> get appBlocProviders =>
       _allAppImplementations.map((final appImplementation) => appImplementation.blocProvider).toList();
+
+  UnifiedSearchBloc get unifiedSearchBloc => UnifiedSearchBloc(
+        _requestManager,
+        _account.client,
+        appImplementations.value.data,
+      );
```

Comment:
I don't think SearchBloc where _appImplementations is empty or null make sense so we should probably jsut make the getter nullable.

Also `appImplementations.value` could be risky. I think we would need to use `appImplementations.valueOrNull` but I didn't 100% check that

Reply:
You are right, but the button only appears if `appImplementations` has a value. Still not that great...

Reply:
I just looked at it and we already check so we could indeed make it nullable or add an assertion or something.
```dart
if (appImplementations.hasData) ...[
]
```